Description: Remove unused build dep and features
 Removed features are used upstream to control build of the embedded
 libmimalloc, but the Debian packages use Debian's libmimalloc.
Last-Update: 2025-03-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,1 +22,0 @@
-default-features = false
@@ -30,3 +22,0 @@
-debug = ["libmimalloc-sys/debug"]
-debug_in_debug = ["libmimalloc-sys/debug_in_debug"]
-default = []
@@ -40,4 +22,0 @@
-local_dynamic_tls = ["libmimalloc-sys/local_dynamic_tls"]
-no_thp = ["libmimalloc-sys/no_thp"]
-override = ["libmimalloc-sys/override"]
-secure = ["libmimalloc-sys/secure"]
